ctrl/ctrl1.go

```diff
@@ -554,10 +571,103 @@ func home(w http.ResponseWriter, r *http.Request) {
 		cmd = exec.Command(binariesPath+"/cleanUP", args...)
 		cmd.Start()
 		cmd.Wait()
+	case "test_list":
+		testlists, err := getStandardTests()
+                if err != nil {
+                        base.Zlog.Infof(err.Error())
+			return
+                }
+		returnData, err := json.Marshal(testlists)
+                if err != nil {
+                        base.Zlog.Infof(err.Error())
+                }
+		w.Write([]byte(returnData))
+	case "test_start":
+		base.Zlog.Infof("Starting OpenBMC Testing")
+		_, tail := ShiftPath(r.URL.Path)
+		path := strings.Split(tail, "/")
+		username := path[1]
+		base.Zlog.Infof("Username:%s", username)
+		input := r.FormValue("testlist")
+		var  tests []string
+		json.Unmarshal([]byte(input), &tests)
+		if len(tests) < 1{
+			base.Zlog.Infof("Empty Tests")
+			w.Write([]byte("Empty test lists"))
+			return
+		}
+		testlist := strings.Join(tests, ",") 
+		base.Zlog.Infof("Testlist:", testlist)
+
+		if contestStartCmd != nil {
+			unix.Kill(contestStartCmd.Process.Pid, unix.SIGTERM)
+			contestStartCmd = nil
+		}
+		var args []string
+		args = append(args, "-p")
+		args = append(args, "8081")
+		args = append(args, "-s")
+		args = append(args, "9")
+		args = append(args, "-t")
+		args = append(args, "disableReconnect=true")
+		args = append(args, binariesPath + "/contestcli")
```

Comment:
might be (not fully sure) interesting to record the session into a screen like that end user can use Page Up / Down to browse logging output